File: ESC.Resturant.Data/Entities/BillTaxes.cs
using ESC.Resturant.Data.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public class BillTaxes
    {
        [Key]
        public int Id { get; set; }
        public int? TaxesId { get; set; }
        public int? BillId { get; set; }
        public double PercentValue { get; set; }
        public double Total { get; set; }
        public virtual Bill Bill { get; set; }
        public virtual Taxes Taxes { get; set; }

    }
}

File: ESAResturant.NEW.web/ClientApp/src/app/admin/bill-invoice/Models/BillTaxes.ts
import { Taxes } from "../../setting/Models/Taxes";
import { Bill } from "./Bill";

export class BillTaxes  {

  constructor() {

    this.id = 0;
    this.billId = 0;
    this.taxesId = 0;
    this.total = 0;
    this.percentValue = 0;

  }
  
  id: number;
  billId: number;
  taxesId: number;
  total: number;
  percentValue: number;
  bill: Bill;
  Taxes: Taxes;
}


